lib/terraform/runner/response.rb

```diff
@@ -0,0 +1,56 @@
+require 'json'
+
+module Terraform
+  class Runner
+    class Response
+      include Vmdb::Logging
+
+      attr_reader :stack_id, :stack_name, :status, :action, :message, :error_message,
+                  :details, :created_at, :stack_job_start_time, :stack_job_end_time
+
+      # @return [String] Extracted attributes from the JSON response body object
+      def self.parsed_response(http_response)
+        data = JSON.parse(http_response.body)
```

Comment:
Question:  can the response body be empty/bad in the event of a 500 error or similar?  I'm wondering if `.body` will always be able to be `JSON.parse` against.